src/elastio-snap.c

```diff
@@ -3262,7 +3277,7 @@ static int snap_trace_bio(struct snap_device *dev, struct bio *bio){
 	unsigned int bytes, pages;
 
 	//if we don't need to cow this bio just call the real mrf normally
-	if (!bio_needs_cow(bio, dev) || memory_is_too_low(dev)) {
+	if (!bio_needs_cow(bio, dev) || memory_is_too_low(dev) || tracer_read_cow_file_state(dev)) {
```

Comment:
I'd like this comment to be updated to reflect reality.  It was written when only `bio_needs_cow` was used but there are actually three conditions.

It's not documented anywhere what `cow_file_state` actually represents or what values it may have.  What does a non-zero value mean?